# Custom scripts for ploidy, mode of reproduction, environmental, and genomic SNP analyses
Custom scripts for ploidy, mode of reproduction, environmental, and genomic SNP (RAD-Seq) analyses in taxonomically complex groups (TCGs), exemplified on the Ranunculus auricomus plant species complex.

The code repository includes the R script used in Karbstein et al. 2021 (https://onlinelibrary.wiley.com/doi/10.1111/mec.15919). Most of the datasets used are available on Figshare (https://doi.org/10.6084/m9.figshare.13352429).

The R script can perform (i) data import of flow cytometric (FC) and flow cytometric seed screening (FCSS) data, (ii) descriptive statistics, (iii) mode of reproduction per ploidy level analyses, (iv) multivariate modeling (path analysis) comprising sexuality (sexual to asexual seeds per population), genome-wide heterozygosity, ploidy level, mode of reproduction, habitat type, and flower features, (v) modeling and mapping of sexuality and genetic diversity across Europe, (vi) area calculations per reproduction mode, and (vii) genetic structure analyses (sNMF).



# If you use the code, please cite: 

Karbstein K, Tomasello S, Hodač L, Wagner N, Marincek P, Barke BH, Pätzold C, Hörandl E. Untying Gordian knots: Unraveling reticulate polyploid evolution by RRS genomic data using the large Ranunculus auricomus species complex. New Phytologist, 2022; 235; 2081-2098. https://doi.org/10.1111/nph.18284
